cognite/client/_api_client.py

```diff
@@ -29,7 +29,11 @@
 from requests.structures import CaseInsensitiveDict
```

Comment:
Looks like all changes in this file are formatting changes. Please put those and any other such changes in a separate PR or undo. These huge PRs are painful enough to review as is 😉

Reply:
Did not notice this. It was a mistake from me. Will fix.

Reply:
Done